scripts/forger/seed/download_seed.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+# This script is used to download and extract the blockchain data from a seed file.
+set -eEuo pipefail
+
+command -v aria2c &>/dev/null || {
+  echo "${FUNCNAME[0]} Error: 'aria2c' is required to run this script, see installation instructions at 'https://github.com/aria2/aria2/releases/tag/release-1.37.0'."
+  exit 1
+}
+
+script_dir="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
+deployment_dir="$(readlink -f "${script_dir}/../")"
+seed_dir="${deployment_dir}/seed"
+env_file="${deployment_dir}/.env"
+
+# shellcheck source=../.env
+source "${env_file}"
```

Comment:
See if you want to check first if env file exist and exit with the message if does not